google/cloud/compute/samples/disk_samples.cc

```diff
@@ -87,7 +87,8 @@ void CreateEmptyDisk(std::vector<std::string> const& argv) {
   }
   //! [END compute_disk_create_empty_disk]
   (argv.at(0), argv.at(1), argv.at(2), argv.at(3),
-   SplitLabelsString(argv.at(4)));
+   ((argv.size() == 5) ? SplitLabelsString(argv.at(4))
```

Comment:
I think now I'd change the "throw Usage" condition to `if (argv.size() != 4 && argv.size() != 5)`.